scripts/export-for-ops.ts

```diff
@@ -146,48 +172,43 @@ const main = async () => {
     return {
       ...readJsonFile(join(templatesBasePath, sourceFile)),
       sourceFile,
+      name: path.parse(sourceFile).name,
     };
   });
 
   const promisedBulkPayload = await Promise.all(
-    jobs.map(async ({ templateId, parameters, endpointName, deviationPercentage, keeperSponsor, requestSponsor }) => {
-      const templateObj = templates.find((template) => template.sourceFile === `${templateId}.json`);
+    jobs.map(async ({ templateId, parameters, deviationPercentage, keeperSponsor, requestSponsor }) => {
+      const templateObj = templates.find((template) => template.templateId === templateId);
 
       const hdNode = verifyAirnodeXpub(apiMetadata.xpub, templateObj.airnode);
       const keeperSponsorPath = deriveKeeperWalletPathFromSponsorAddress(keeperSponsor);
       const apiProviderUpdateRequester = hdNode.derivePath(keeperSponsorPath).address;
 
-      const api3HdNode = ethers.utils.HDNode.fromExtendedKey(api3Xpub);
-      const api3AirnodeAddress = api3HdNode.derivePath('0/0').address;
       const encodedParameters = abi.encode(parameters);
       const beaconId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [templateId, encodedParameters]);
       const beaconDescriptor = {
         templateId,
+        templateName: templateObj.name,
         parameters: templateObj.parameters,
         decodedParameters: templateObj.decodedParameters,
         beaconId: beaconId,
         chains: await Promise.all(
           templateObj.chains.map(async (chainName: string): Promise<ChainDescriptor> => {
             return {
               name: chainName,
+              sponsor: { address: requestSponsor },
               apiProviderAirkeeperDeviationPercentage: parseFloat(deviationPercentage),
               api3AirkeeperDeviationPercentage: parseFloat(deviationPercentage) * 2,
-              apiProviderUpdateRequesterMetadata: { address: apiProviderUpdateRequester },
-              apiProviderAirnodeFulfillerMetadata: {
+              apiProviderAirkeeperSponsor: { address: apiProviderUpdateRequester },
+              apiProviderAirnodeSponsor: {
```

Comment:
The `Sponsor` addresses should be different from the `SponsorWallet` addresses. I think there should be both `Sponsor` and `SponsorWallet` Addresses in the final ops that is exported.

Reply:
Great, I'll add them verbatim from `airkeeper.json`

Reply:
Are we saying that the final ops that is exported should then have all 6 addresses?
sponsor=>sponsorWallet
apiProviderAirkeeperSponsor=>apiProviderAirkeeperSponsorWallet
api3AirkeeperSponsor=>api3AirkeeperSponsorWallet

I only ask because sponsor wallets can always be derived by calling deriveSponsorWalletAddress with the airnode metadata plus the sponsor address.

Reply:
I'm flexible. My primary goal is to get this schema into a stable state so that I can make associated changes everywhere else (telemetry collector/notifiers/wallet top-ups/database).